elixir/plan.sh

```diff
@@ -1,14 +1,15 @@
 pkg_origin=core
 pkg_name=elixir
-pkg_version=1.3.2
+pkg_version=1.3.4
+pkg_description="A dynamic, functional language designed for building scalable and maintainable applications. Elixir leverages the Erlang VM, known for running low-latency, distributed and fault-tolerant systems, while also being successfully used in web development and the embedded software domain."
+pkg_upstream_url=http://elixir-lang.org
 pkg_maintainer="Shane Sveller <shane@shanesveller.com>"
-pkg_license=('apachev2')
-pkg_source=https://github.com/elixir-lang/elixir/archive/v1.3.2.tar.gz
-pkg_shasum=be24efee0655206063208c5bb4157638310ff7e063b7ebd9d79e1c77e8344c4b
-pkg_deps=(core/busybox core/cacerts core/coreutils core/openssl core/erlang/18.3)
-pkg_build_deps=(core/busybox core/cacerts core/coreutils core/make core/openssl core/erlang/18.3)
+pkg_license=('Apache-2.0')
+pkg_source=https://github.com/elixir-lang/elixir/archive/v1.3.4.tar.gz
```

Comment:
Changes made. I've also submitted to change the `pkg_maintainer` to Habitat Maintainers